Add field specification for EVSE identifier

RoiEXLab opened the pull request:
### Description, Motivation & Context
The [EVSE](https://wiki.openstreetmap.org/wiki/Key:ref:EU:EVSE) identifier is currently used to uniquely identify charging ports on charging stations for electric vehicles. Even though wide adoption of the tag is not there yet (and it's sometimes used incorrectly) the [wiki page](https://wiki.openstreetmap.org/wiki/Key:ref:EU:EVSE) already documents the format pretty well. Often public charging infrastructure has more than one plug (usually 2, each with a unique id), so I changed the regex to allow semicolon-separated lists of ids (which seems to be a popular notation: https://taginfo.openstreetmap.org/keys/ref%3AEU%3AEVSE#values).

#### Caveats
The notation of the id is specified in a very lenient way, allowing many strings that represent the same id. The wiki page presents a canonical notation, but it could be confusing to users because on actual charging stations they might use spaces as separators, or use more than two separators when looking at them in person.

For `man_made=charge_point` and `amenity=charging_station` the `ref` key is always recommended. Since the `EVSE` identifier only applies to EU member states I added this new field for the preset, but the common `ref` still remains, even though it is kinda redundant in these places.

### Links and data

**Relevant OSM Wiki links:**
- https://wiki.openstreetmap.org/wiki/Key:ref:EU:EVSE

**Relevant tag usage stats:**
- Currently 17037 total uses (according to https://taginfo.openstreetmap.org/)

### Checklist and Test-Documentation Template

## Test-Documentation

### Preview links & Sidebar Screenshots

![Screenshot amenity=charging_station](https://github.com/user-attachments/assets/58d7a8b5-57da-49a1-b4a2-26b378670c4f)
[link](https://pr-1752--ideditor-presets-preview.netlify.app/id/dist/#locale=en&map=18.80/48.82406/2.32435&disable_features=boundaries&background=fr.ign.bdortho&id=n12415996660)

![Screenshot man_made=charge_point](https://github.com/user-attachments/assets/a1fc4c44-6249-4c68-b3c8-e13f84e6154e)
[link](https://pr-1752--ideditor-presets-preview.netlify.app/id/dist/#locale=en&map=19.84/48.81045/2.48160&disable_features=boundaries&background=fr.ign.bdortho&id=n13030359201)


<!-- Use the preview to find examples, select the feature in question and **copy this link here**.
     Find examples of nodes/areas. Find examples with a lot of tags or very few tags. – Whatever helps to test this thoroughly.
     Add relevant **screenshots** of the sidebar of those examples. -->

<!-- FYI: What we will check:
     - Is the [icon](https://github.com/ideditor/schema-builder/blob/main/ICONS.md) well chosen.
     - Are the fields well-structured and have good labels.
     - Do the dropdowns (etc.) work well and show helpful data. -->

### Search

Does not apply, only applicable to EU member states and there it is pre-selected for all relevant feature types.

### Info-`i`

![Info-`i`](https://github.com/user-attachments/assets/7e724d79-3426-41b1-bfeb-4d5f29272c12)

### Wording

- [x] American English
- [x] `name`, `aliases` (if present) use Title Case
- [x] `terms` (if present) use lower case, sorted A-Z

FloEdelmann commented on data/fields/ref/EU/EVSE.json:
```diff
@@ -0,0 +1,16 @@
+{
+    "key": "ref:EU:EVSE",
```
The part after `ref:` is often uppercase, as it refers to the country (DE, SE, EU, ...) instead of the language (de, sv, ...).

RoiEXLab commented:
Anything else I need to do to a) get the workflows approved and b) get this thing merged in the end?

FloEdelmann commented on data/fields/ref/EU/EVSE-EU.json:
```diff
@@ -0,0 +1,18 @@
+        "electrical vehicle supply equipment",
+        "evse",
+        "charging identifier",
+        "charger identifier"
```
Please sort these terms alphabetically:

```suggestion
        "charger identifier",
        "charging identifier",
        "electrical vehicle supply equipment",
        "evse"
```

RoiEXLab commented:
@tyrasd Thank you for linking the related issue, I had no idea this was happening at the same time.

@matkoniecz I corrected the spelling, thanks for digging up an official source. I didn't quite follow your suggestion, but I added "ID" at the end which should hopefully make things clearer. (Will update the screenshots in the original post once the CI lets me, GH is so annoying with these limitations for first-time contributors, but oh well).

> Single charging point can have multiple sockets each with own code. Would it make sense to have some support for such multivalues?

I don't quite get what you mean by "multivalues". According to https://taginfo.openstreetmap.org/keys/ref%3AEU%3AEVSE#values it is already very common practice to simply separate multiple values using a semicolon. I don't think the tagging schema has a way to represent that unfortunately.
